src/rules/converters/quotemark.ts

```diff
@@ -0,0 +1,32 @@
+import { RuleConverter } from "../converter";
+
+export const convertQuotemark: RuleConverter = tslintRule => {
+    const notices: string[] = [];
+    const ruleArguments: any[] = [];
```

Comment:
`any` is kind of a dangerous bad kid in TypeScript land. It removes a lot of type safety from its operations. Let's switch to the more specific `string[]`:

```suggestion
    const ruleArguments: string[] = [];
```

Reply:
I can't use string[] as avoidEscape rule is an object, but I created the type below that is now used as the array type.

```ts
type QuotemarkRule = string | { avoidEscape: true }
const ruleArguments: QuotemarkRule[] = [];
```